### PR TITLE
Update role with lint ansible and test scenario on molecule

### PR DESCRIPTION
**Pull request by StartloJ**
### Problem finds
When I run this role more 1 times for update some config after set up jenkins success, ansible report error with tasks in post-file missing [ **/var/lib/jenkins/secrets/initialAdminPassword** ]
Then some test scenario ansible error with missing dependencies like gpg. In my case I always need git dependencies, so it can prepared in installation phase.

### Solving
Add register in tasks install jenkins and wait_for initialAdminPassword file. 
Next check if they were changed, then tasks get password file can run, else it's should skipped this tasks.
```ansible
- name: Register variable to check Jenkins is installed (Debian/Ubuntu)
  when: ansible_os_family == "Debian"
  set_fact:
    task_install_changed: jenkins_installed.changed
```
```ansible
- name: Retry for expose initial password
  when: task_install_changed | bool or not find_init_token.failed
  block:
    - name: Get initial admin password
      command: cat /var/lib/jenkins/secrets/initialAdminPassword
      register: jenkins_initial_admin_password
      changed_when: false
    - name: Initial admin password is
      debug:
        msg: "{{ jenkins_initial_admin_password.stdout }}"
```
Then add tasks for prepared some dependencies
```ansible
- name: Ensure remotely host has require dependencies
  when: ansible_os_family == "Debian"
  apt:
    name: "{{ standard_dep }}"
    state: present
    update_cache: yes
```

### Add ansible role tester
Duplicated test case from prev. roles and edit details config specified with jenkins-roles.
Now I can run test in only ubuntu 18.04 , python 3.7 and ansible 2.5-2.8

### Refactor code
Clear useless script and refactor some tasks with suggest from ansible lint such as
1. change task to use command from shell module.
```ansible
-  name: Get initial admin password
   command: cat /var/lib/jenkins/secrets/initialAdminPassword
   register: jenkins_initial_admin_password
   changed_when: false
```
2. clean unnecessary whitespace in yaml file.
3. add changed_when for tasks control in command or shell module.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1 +1,5 @@
----
\ No newline at end of file
+---
+standard_dep: #[]
```
Comment (3d-pro) [inline defaults/main.yml]: Update variables name with `jenkins_` prefix

File: tasks/jenkins_pre.yml [hunk 2]
```diff
@@ -1,4 +1,11 @@
 ---
+- name: Ensure remotely host has require dependencies
+  when: ansible_os_family == "Debian"
+  apt:
+    name: "{{ standard_dep }}"
```
Comment (3d-pro) [inline tasks/jenkins_pre.yml]: Update variables name with `jenkins_` prefix

**Comment by 3d-pro** [comment]
added comments on variable naming.

**Comment by StartloJ** [comment]
> added comments on variable naming.

I following on your commented to add text explain variable in default/main.yml and remove test scenario with python27 on molecule.

**Review by 3d-pro — APPROVED** [review]

**Pull request merged**